scripts/linters/test_files/invalid_iit_fit.ts

```diff
@@ -60,7 +60,8 @@ describe('Collection node object factory', () => {
     expect(summaryObject).not.toEqual(
       collectionNode.getExplorationSummaryObject());
   });
-// fit is not allowed to use.
+
+  // fit is not allowed to use.
```

Comment:
The backend tests are still failing. I think reverting this change will fix them.